Makefile

```diff
@@ -44,9 +44,9 @@ apicast-source: ## Create Docker Volume container with APIcast source code
 	docker create --rm -v /opt/app-root/src --name $(COMPOSE_PROJECT_NAME)-source $(IMAGE_NAME) /bin/true
 	docker cp . $(COMPOSE_PROJECT_NAME)-source:/opt/app-root/src
 
-
-busted: dependencies $(ROVER) ## Test Lua.
-	@$(ROVER) exec bin/busted
+BUSTED_FILES ?=
+busted: dependencies $(ROVER) ## Test Lua. To test a specific file: make busted BUSTED_FILES=spec/resty/testname.lua 
```

Comment:
I'm not sold. I don't think this needs to be documented. 
If you are skilled enough to want to run just one test you'll just run `bin/busted` without the makefile.

Reply:
Fair enough :)

Reply:
done in 72f9a669a6fae2a9be05f3c9faa03060dc1ee5e7